src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
```

Comment:
```suggestion
export const PostInfo: React.FC<Props> = (props) => {
  const { post } = props;
```

Reply:
i got this point. is it best practice? i have seen during lectures mentor use object destructuring  when we pass the argument. i do now understand that in that was if we decide to pass more stuff, we won't be able to asses it from props because we have restructured it. thank you!